src/scripts/components/Header.jsx

```diff
@@ -61,30 +61,33 @@ export default class Header extends React.Component {
     return (
       <div className="row header__row">
         <Errors />
-        <h1 className="header__h1 col s11">
-          <img
-            src="/assets/images/stamp.png"
-            className="header__stamp"
-            alt="Resound Store logo"
-          />
-          <Link to="/" className="header__link">
-            store
-          </Link>
-        </h1>
-        <div className="header__actions col s1">
-          <button
-            className={loggedIn ? "hidden" : "header__button"}
-            onClick={this.login}
-          >
-            log in
-          </button>
-          <button
-            className={loggedIn ? "header__button" : "hidden"}
-            onClick={this.logout}
-          >
-            log out
-          </button>
-        </div>
+        {this.state.loggedIn &&
+          <div>
+            <h1 className="header__h1 col s11">
+              <img
+                src="/assets/images/stamp.png"
+                className="header__stamp"
+                alt="Resound Store logo"
+              />
+              <Link to="/" className="header__link">
+                store
+              </Link>
+            </h1>
+            <div className="header__actions col s1">
+              <button
+                className={loggedIn ? "hidden" : "header__button"}
+                onClick={this.login}
+              >
+                log in
+              </button>
+              <button
+                className={loggedIn ? "header__button" : "hidden"}
+                onClick={this.logout}
+              >
+                log out
+              </button>
```

Comment:
I find it curious that this part of the template appears to only show up if `this.state.loggedIn`, but you have conditionals for the button class names.  Do they still get inserted if `this.state.loggedIn` is false?  I'm assuming the answer is no because they markup comes after `&&`, but I could be mistaken.